src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -700,7 +700,7 @@ services:
     tags:
       - { name: form.type }
 
-  from.type.yes_and_no_choice:
+  form.type.yes_and_no_choice:
```

Comment:
Possible break change, can you please update the PR description ?
To avoid BC we can add service alias, and [deprecate](https://symfony.com/doc/current/service_container/alias_private.html#deprecating-services) the old service ?

Reply:
Is this really a BC break as 8.0 is not out yet ?

Reply:
As it's was introduced inside the 1.7.6, yes, it is.

Reply:
ok,  but 1.7.6 is not targeted by this PR (based on 8.0.X), so it does not change anything in this version. 8.0.x allows BC breaks from 1.7.X.X as it is a new major.

Why is a deprecation needed on a 8.0.X ?

Reply:
the rule is: first deprecate, then remove, so that developers can prepare for the changes